tools/debian/cockpit-ws.postinst

```diff
@@ -13,3 +13,6 @@ fi
 if [ -d /run/systemd/system ] && [ -n "$2" ]; then
     deb-systemd-invoke try-restart cockpit.service >/dev/null || true
 fi
+
+# Link to inactive message if link doesn't exist
+ln -s /etc/cockpit/inactive.issue /etc/cockpit/cockpit.issue || true
```

Comment:
The `#DEBHELPER#` macro will be replaced with code that automatically enables and starts `cockpit.socket` on "real" systems, so this would always fail. And in chroots, where services are not started automatically (or manually), we should not do the issue magic either, as the shown information would just be plain wrong. So I think this should just be dropped.